docs/api/resource.rst

```diff
@@ -1,28 +1,275 @@
+.. _resource-endpoints:
+
 ##################
 Resource endpoints
 ##################
 
-.. _resource-endpoints:
+GET /buckets/syncto/collections/{collection}/records
```

Comment:
If we do it here, we should probably as well in ``cliquet_docs/api/resource.rst``